open_schools_platform/tasks/tasks.py

```diff
@@ -6,7 +6,7 @@
 
 
 @app.task
-def send_mail_to_new_user_with_celery(subject, html_kwargs, from_email, to):
+def send_mail_to_new_user_with_celery(subject, html_kwargs, from_email, to, template='new_user_invite_mail_form.html'):
```

Comment:
it is better to delete ='new_user_invite_mail_form.html'
change function in other place by insert 'new_user_invite_mail_form.html'